### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by spartyboy** [beyondcode/laravel-debugbar-companion#7]
companion keeps showing There are no current request yet. and i have configured the config file and restarted server

**Comment by leandrodiogenes** [comment]
Same here.

**Comment by jbxonline** [comment]
Yep, same issue. I'm wondering if, in my case, this is a docker issue?
Does this work with containers?

**Comment by DavyDeCoster** [comment]
@jbxonline In a container, you would need to change the host IP-address. Is it possible to change your config in the `config/debugbar.php` like this and fill in the host ip address:

```
'storage' => [
    'enabled'    => true,
    'driver'     => 'socket', // redis, file, pdo, custom
    'path'   => storage_path('debugbar'),
    'connection' => null,
    'hostname' => 'YOUR HOST IP ADDRESS HERE',
    'port' => 2304
],
```

**Comment by lubop** [comment]
Hello, 
same issue here.
Changed driver in config/debugbar.php to 'socket' (instead of 'app' proposed in client),
and after debugging it creates and send data to socket, but the client (windows) is not processing the payload. 

After enabling developers toolbar in windows client, it shows this:
![image](https://user-images.githubusercontent.com/2005577/104195777-55806b00-5423-11eb-96d5-371213960c32.png)

seems like this is the problem:
![image](https://user-images.githubusercontent.com/2005577/104196324-eeaf8180-5423-11eb-8adf-bcc21c43547c.png)

**Comment by DavyDeCoster** [comment]
@lubop This is fixed by commit 1f5fa4b34b1f2f9c21668cc0e6afcab7e897854e, but @mpociot didn't create a release yet. If you want, you can use the current master-branch.

**Comment by lubop** [comment]
@DavyDeCoster thanks, that was the problem.
Managed to build release from current master branch and now it works...

**Comment by gpibarra** [comment]
According to the documentation, the class "SocketStorage" says that it is in master repo of DebugBar, but it is also in the stable version.
Version 0.1.0
OS: Win10 x64
DebugBar version: 3.5.2

I change file "config / debugbar.php" to "driver=>socket". (When starting the app Companion, it says to put "driver => app" on it, but it must be "soket").
The info is sent correctly to the socket (I have tried listening to the port with [https://packetsender.com/](https://packetsender.com) and the info is sent from the Laravel app).

The Companion app never shows the information.

**Comment by spartyboy** [comment]
@DavyDeCoster how do i then compile the code to an electron app executable by windows

**Comment by DavyDeCoster** [comment]
@spartyboy install all dependencies using Yarn and then run `yarn run build:win` should build a executable. If not, try `yarn run publish:win`

**Comment by leandrodiogenes** [comment]
yea, indeed we need to wait for a new release, master branch it works fine.

**Comment by Claw-Hammer** [comment]
Same here, waiting for a new release

**Comment by Claw-Hammer** [comment]
> @spartyboy install all dependencies using Yarn and then run `yarn run build:win` should build a executable. If not, try `yarn run publish:win`

Thanks @DavyDeCoster, i was able to build the app from the master branch, it's working now

**Comment by spartyboy** [comment]
> @spartyboy install all dependencies using Yarn and then run `yarn run build:win` should build a executable. If not, try `yarn run publish:win`

$ yarn electron:build -- --win  --x64 --ia32 -p always
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
$ vue-cli-service electron:build --win --x64 --ia32 -p always
'vue-cli-service' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
here is the error i get on trying to build it

**Comment by Claw-Hammer** [comment]
@spartyboy i can send you the .exe installer, just let me know

**Comment by spartyboy** [comment]
> @spartyboy i can send you the .exe installer, just let me know

pls do

**Comment by Claw-Hammer** [comment]
For all those who need it, in the following link you can download the installer for Windows: 

https://www.4shared.com/s/fW17WRXY1iq

This build was made by myself, no virus, no scam, no BS....just the debugbar companion ... !

**Comment by keithbrink** [comment]
@mpociot Any chance of getting a fixed release for Mac?

**Comment by t-raven** [comment]
for anyone who - like me - googled this issue when looking how to set up debugbar companion to work with docker:

in `docker-compose.yml` (add to container with laravel):
```
extra_hosts:
    - 'host.docker.internal:host-gateway'
```

in `config/debugbar.php` set
```php
'storage' => [
    'hostname' => 'host.docker.internal'
    (...)
]
```

**Comment by truesteps** [comment]
@mpociot could you please make a release for mac? Still doesn't work unfortunately. This issue resides [here](https://github.com/beyondcode/laravel-debugbar-companion/issues/7#issuecomment-757999274)
